src/test/java/PeselValidatorTest.java

```diff
@@ -0,0 +1,57 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class PeselValidatorTest {
+
+    private  PeselValidator validator = new PeselValidator();
+
+    @Test
+    public void testValidadPesel() {
+
+       assertTrue(validator.validate("59010396898"));
+
+    }
+
+    @Test
+    public void testPeselWhichIsTooShort() {
+
+        assertThrows(NumberFormatException.class, () -> validator.validate("5901039689"));
+
+    }
+
+
+    @Test
+    public void testPeselWithWrongMonth() {
```

Comment:
This is the same test as the first one - testValidPesel() just with different name, so this is redundant and can be removed. Probably you didn't follow basic rule of TDD: red-green-refactor, and this test was from the beginning a green one.